src/sentry/lang/javascript/processing.py

```diff
@@ -12,6 +13,10 @@
 
 logger = logging.getLogger(__name__)
 
+# Matches "app:", "webpack:",
+# "x:" where x is a single ASCII letter, or "/".
+NON_BUILTIN_PATH_REGEX = re.compile(r"(app|webpack|[a-zA-Z]):|/")
```

Comment:
```suggestion
NON_BUILTIN_PATH_REGEX = re.compile(r"^((app|webpack|[a-zA-Z]):|/)")
```

We have to match the start, otherwise any `/` in the middle would match this.

Reply:
[According to the Python docs](https://docs.python.org/3/library/re.html#re.Pattern.match), `match` only matches at the start. I think it's probably still worth it to change the regex for clarity though.

Reply:
Oh interesting, I didn’t know that either. Do we have any kind of tests for this?

Reply:
Do we at least have an e2e test that uses the `node` platform with non-http paths?